### PR TITLE
Utilize clip_values for pytorch

### LINKED ISSUE #243: Utilize clip_values for all baseline datasets
This will need to be done carefully so as to match the preprocessing functions for a given baseline model. `clip_value` is necessary to specify in an ART classifier so that attacks generate examples within bounds.

### PR DESCRIPTION
**Pull request by seanpmorgan**
Closes #243

### CODE REVIEW

File: armory/baseline_models/pytorch/imagenet.py [hunk 1]
```diff
@@ -18,74 +16,32 @@ def resnet50(pretrained=True):
...
     model = resnet50(**model_kwargs)
     wrapped_model = PyTorchClassifier(
         model,
-        loss=nn.CrossEntropyLoss(),
-        optimizer=None,
+        loss=torch.nn.CrossEntropyLoss(),
+        optimizer=torch.optim.Adam(model.parameters(), lr=0.003),
         input_shape=(224, 224, 3),
         **wrapper_kwargs,
+        clip_values=(0.0, 1.0)
```
Comment (davidslater) [inline armory/baseline_models/pytorch/imagenet.py]: I don't think the values are in (0.0, 1.0) after normalization.

Comment (davidslater) [reply armory/baseline_models/pytorch/imagenet.py]:
```
        clip_values=(0 - np.array(means)])/np.array(stds), (1 - np.array(means))/np.array(stds))
```

Comment (seanpmorgan) [reply armory/baseline_models/pytorch/imagenet.py]: Thanks. Changed
